### DISCUSSION TITLE
Is it possible to add several models for Openrouter?

### DISCUSSION THREAD

**Discussion by alesha-pro** [janhq/jan#4153] [Get Help]
I have reread the documentation, searched through a bunch of Issues and still don't understand. Is it possible to add several models for Openrouter with the possibility to choose between them directly in the interface. It seems to be such an obvious functionality that it should be out of the box, but I still don't get it. 
Can you please tell me if this is possible, and if so, how?

**Comment by Gargantubrain** [comment]
Please add this ability, I really can't use this software without this feature! Is there any way to at least duplicate the OpenRouter extension more than once, so we can use the only method currently provided with it to enter a hard-coded model like "google/gemini-pro-1.5", so that we can at least add multiple OpenRouter models, one for each Model we need? Note this would also require the ability to rename the extension to match the model, it would be useless if we just see "OpenRouter" 3 times in the list and couldn't tell which is which. Thank you!
